riklet/src/cli/mod.rs

```diff
@@ -0,0 +1,66 @@
+pub mod config;
+pub mod function_config;
+
+use clap::{value_parser, Parser};
+use std::net::Ipv4Addr;
+
+/// The configuration of the riklet.
+#[derive(Debug, Clone, Parser)]
+#[command(name = "Riklet", version, about)]
+pub struct CliConfiguration {
+    /// The path to the Riklet configuration file. If the file not exists, it will be created.
+    #[arg(
+        short,
+        long,
+        default_value = "/etc/riklet/configuration.toml",
+        help = ""
```

Comment:
A local configuration file would be preferred, as we don't want to "pollute" file systems.
 
```suggestion
        short,
        long,
        default_value = "riklet.toml"
```

Reply:
I don't want to mess with this configuration (I'm not the author). I think that the default configuration is set to this path to it has to stay like this for the moment.

Reply:
ok